notebooks/code_samples/customization/customizing_metrics_with_output_templates.ipynb

```diff
@@ -546,13 +548,13 @@
             "metadata": {},
             "outputs": [],
             "source": [
-                "## Replace this placeholder with the code snippet from your own project ##\n",
+                "## Replace with your code snippet\n",
                 "\n",
                 "vm.init(\n",
-                "  api_host = \"https://api.prod.validmind.ai/api/v1/tracking\",\n",
-                "  api_key = \"...\",\n",
-                "  api_secret = \"...\",\n",
-                "  project = \"...\"\n",
+                "    api_host=\"https://api.prod.validmind.ai/api/v1/tracking\",\n",
```

Comment:
This looks to have been introduced by my linter — Python uses 4 spaces for indenting, as recommended by the PEP 8 style guide.